jac-uk/apply
# #1048 Additional Welsh translation for exercise 172

## What's included?
Closes #1048 

Note: introduce two helper functions (`getDataWelshId` and `getDataWelsh`) to deal with the multi-version Welsh translation of `Yes` and `No`. It would be appreciated if there is any suggestion/feedback for this way.

Update: the Welsh translation of the part will be provided by noon on 6 September

![image](https://github.com/jac-uk/apply/assets/79906532/2d9ea4c2-8c7f-4c5f-9c58-06acdd261fcc)


## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Apply for this vacancy in Welsh:
    Dev: https://jac-apply-develop--pr1050-feature-1048-additio-7vhg4t4c.web.app/vacancy/2M7yxJySfXpINFwvhnnW
    Staging: https://jac-apply-staging--pr1050-feature-1048-additio-a8wbhbqy.web.app/vacancy/bvRP44PULSsIKaJN1yr2

2. Check if the Welsh translation works correctly.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:STAGING
_can be OFF, DEVELOP or STAGING_
